taxonomies/categories.txt

```diff
@@ -23295,8 +23295,9 @@ ciqual_food_name:fr:Biscuit sec ou tuile, aux amandes
 fr:tuile aux amandes
 
 <en:Biscuits
+<en:Drop cookies
```

Comment:
in fact we should not add <en:Drop cookies as it's a special type of cookies.

the "en:biscuits" entry already has cookies as a synonym:

en:Biscuits, biscuit, cookies, cookie

Reply:
So we can't fix issue #3961 /it is already fixed?

Reply:
I think it's fixed in fact:

https://world.openfoodfacts.org/category/oatmeal-cookies

-> it is in "biscuits", which is the main name for cookies in English.

We made a distinction between "cookies" and "drop cookies".  In France, "cookies" are only "drop cookies". But in the US / UK, they can be any type of biscuits.

Reply:
But we should merge your other changes.

Reply:
Okay, good